src/libraries/Native/build-native.sh

```diff
@@ -115,6 +118,20 @@ elif [[ "$__TargetOS" == iOS ]]; then
         echo "Error: Unknown iOS architecture $__BuildArch."
         exit 1
     fi
+elif [[ "$__TargetOS" == tvOS ]]; then
+    __CMakeArgs="-DCMAKE_SYSTEM_NAME=tvOS $__CMakeArgs"
+    if [[ "$__BuildArch" == x64 ]]; then
+        # set default tvOS simulator deployment target (8.0 is the minimum supported by Xcode 11)
+        # keep in sync with src/mono/Directory.Build.props
+        __CMakeArgs="-DCMAKE_OSX_SYSROOT=appletvsimulator -DCMAKE_OSX_DEPLOYMENT_TARGET=8.0 -DCMAKE_OSX_ARCHITECTURES=\"x86_64\" $__CMakeArgs"
```

Comment:
we can pull out `-DCMAKE_OSX_DEPLOYMENT_TARGET` and set it to `9.0` for both device and simulator, that seems to be what we currently support in mono/mono.

Reply:
also the src/libraries are missing a bunch of checks for tvOS right now. I'd recommend reviewing each usage of `CLR_CMAKE_TARGET_IOS` and `TARGET_IOS`

Reply:
@directhex this wasn't resolved yet.